addons/weather/XEH_postInit.sqf

```diff
@@ -1,10 +1,11 @@
 #include "script_component.hpp"
 
 GVAR(WindInfo) = false;
+
 ["ACE3 Common", QGVAR(WindInfoKey), localize LSTRING(WindInfoKeyToggle),
 {
     // Conditions: canInteract
-    if !([ACE_player, ACE_player, []] call EFUNC(common,canInteractWith)) exitWith {false};
+    if !([ACE_player, ACE_player, ["isNotDragging", "isNotCarrying", "isNotSitting", "isNotInside"]] call EFUNC(common,canInteractWith)) exitWith {false};
```

Comment:
See discussion in https://github.com/acemod/ACE3/pull/8369 for "isNotInside" exception. Maybe only allow it if the unit is in a static weapon or on foot?

Reply:
Yeah, I thought this was discussed in the past and of course it makes no sense for it to be displayed in closed vehicles. Turns out the current implementation also shows outside wind while turned in in `Tank` inheritors 🤔

Reply:
Could just keep isNotInside exception and then `if !(objectParent ACE_Player isKindOf "StaticWeapon") exitWith {false};`

Reply:
Not much change, but check above should be done after canInteractWith with "isNotInside" exception

Reply:
Adding `"isNotInside"` to this check breaks it.
```sqf
{!([ACE_player, ACE_player, ["notOnMap", "isNotDragging", "isNotCarrying", "isNotSitting"]] call EFUNC(common,canInteractWith)) && !(objectParent ACE_Player isKindOf "StaticWeapon")}
```
With it, `canInteractWith` is always false when in a vehicle, causing the second condition to not have any effect and never hiding wind info.

Reply:
> also needs handling for objNull

You mean that `objectParent ACE_Player` could return `<NULL-object>`?
`objectParent ACE_Player isKindOf "StaticWeapon"` while on foot turns into `<NULL-object> isKindOf "StaticWeapon"`, which returns `false`.